crates/oxc_ast/src/ast/js.rs

```diff
@@ -597,7 +597,7 @@ pub struct PrivateFieldExpression<'a> {
 pub struct CallExpression<'a> {
     #[serde(flatten)]
     pub span: Span,
-    pub callee: Expression<'a>,
+    pub callee: Box<'a, Expression<'a>>,
```

Comment:
This change doesn't make sense to me. It introduces indirection to save 8 bytes. And it's not really saving any bytes anyway as the `Expression` is still stored in arena, just in a separate allocation - in fact overall it *costs* an extra 8 bytes for the additional pointer.